components/radial-menu/README.md

```diff
@@ -147,6 +147,18 @@ when an item is selected to the `data-select-event-name` attribute.
 A basic approach which allows executing code through the event and the code or
 the attached functions are decoupled from the component itself.
 
+#### Getting the selected element
+
+To retrieve the currently selected item whenever an item is selected, you need to listen for the custom event that is emitted when a selection is made. You can do this by adding an event listener for the event specified in the `data-select-event-name attribute`.
+
+```javascript
+radialMenu.addEventListener('itemSelected', (event) => {
+    const selectedItem = radialMenu.selectedItem;
+})
+```
+
```

Comment:
Show also how to add the event e.g 
```
<gameface-radial-menu
	data-select-event-name="itemSelected"
	data-open-key-code="SHIFT">
</gameface-radial-menu> 
```

Or use the events from the attribute set in the begining.